### PR TITLE
Add new SDKs, other updates

### PR DESCRIPTION
**Pull request by TomWFox**
- [x] remove some misplaced community projects
- [x] add the flutter, dart & swift SDKs to the front page
- [x] remove flutter & swift from other parts of the front page
- [x] reorganise community projects
- [x] remove redundant JS files
- [x] Rename front page title to from `Parse + Open Source = ❤️` to `Parse Platform`
- [x] Remove `meta` GH projects from the other section
- [x] Changed `iOS + OS X + tvOS` to `Objective-C`

### CODE REVIEW

**Comment by TomWFox** [comment]
Btw this branch isn't actually deployed, GH Pages seemed to be getting stuck after renaming the default branch so I just deployed this one then switched back.

**Comment by TomWFox** [comment]
@phillwiggins @fischerscode @RodrigoSMarques - let me know what you think of the flutter & dart blocks...

Screenshot 2021-01-26 at 14 13 58

**Comment by RodrigoSMarques** [comment]
Hi @TomWFox 
I agree. Very good.

File: _includes/nav.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
             <img src="/img/logo.svg" alt="Parse logo" class="logo">
         </a>
         <ul>
-            <a href="//docs.parseplatform.org" target="_blank"><li>Docs</li></a>
+            <a href="https://docs.parseplatform.org" target="_blank"><li>Docs</li></a>
```
Comment (davimacedo) [inline _includes/nav.html:7]: Maybe do the same for the other links?

**Review by davimacedo — DISMISSED** [review]
LGTM!

**Pull request merged**